src/Node/DatetimeRange.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Gdbots\QueryParser\Node;
+
+final class DatetimeRange extends Range
+{
+    const NODE_TYPE = 'datetime_range';
+    const COMPOUND_NODE = true;
+
+    /**
+     * DatetimeRange constructor.
+     *
+     * @param Datetime|null $lowerNode
+     * @param Datetime|null $upperNode
+     * @param bool $exclusive
+     */
+    public function __construct(Datetime $lowerNode = null, Datetime $upperNode = null, $exclusive = false)
+    {
+        parent::__construct($lowerNode, $upperNode, $exclusive);
+    }
+
+    /**
+     * @return Date
```

Comment:
Hi, sorry for the late reply. I've been quite busy. I updated the doc blocks.
I don't have experience with unit testing. I can maybe have a look at it later, but if you want to help me with that, would be great.
The query builders should work with the new node type, I think.
I'm actually not using the library anymore, because it lacks some more functionalities that I needed (combination of boolean operations, wildcards in the beginning of the search string). But thanks a lot for sharing this library. Nice work!